cmake/fmtlib.cmake

```diff
@@ -1,20 +1,5 @@
-include(FetchContent)
-
-FetchContent_Declare(
-	fmtlib
-	PREFIX "${PROJECT_BINARY_DIR}/deps"
-	DOWNLOAD_DIR "${PROJECT_SOURCE_DIR}/deps/downloads"
-	DOWNLOAD_NAME fmt-9.1.0.tar.gz
-	URL https://github.com/fmtlib/fmt/archive/9.1.0.tar.gz
-	URL_HASH SHA256=5dea48d1fcddc3ec571ce2058e13910a0d4a6bab4cc09a809d8b1dd1c88ae6f2
+add_subdirectory(
+	${CMAKE_SOURCE_DIR}/deps/fmtlib
+	EXCLUDE_FROM_ALL
+	SYSTEM
```

Comment:
``option(FMT_SYSTEM_HEADERS "" ON)`` before ``add_subdirectory`` should have the same effect for newer fmtlib and won't require us to bump the minimum cmake version, so let's just do that :-).

Reply:
I added this already in https://github.com/ethereum/solidity/pull/15195.